magento/adobe-stock-integration
# [WIP] #1515: Add keywords field to the edit image description slide panel

### Description (*)
Added multiselect field for keywords and adjusted Edit Details fieldset DOM to make use of native css form styles. 

**Things to do:** 
- Add/Delete functionality for the Keywords field are not yet working. Based on my investigation, javascript click events are not working since these multiselect elements and classes are rendered dynamically upon opening of slide panel instead of being loaded upon page load.
- Keyword options are also static at the moment.

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1515: Add keywords field to the edit image description slide panel


### Manual testing scenarios (*)

1. Login to Admin panel
2. Go to Content > Media > Media Gallery
3. Select an image, click the "three dots" icon, then click **Edit**
4. Check field labeled "Keywords" at the bottom of the form.
<img width="1677" alt="Screen Shot 2020-07-07 at 11 31 59 PM" src="https://user-images.githubusercontent.com/47961795/86806573-9336bc80-c0ab-11ea-8283-adffb02484b2.png">
